### PR TITLE
Fix mozilla/thimble.webmaker.org#1170 - Track date created and updated for published projects

### PR DESCRIPTION
**Pull request by gideonthomas**
Renamed branch for this PR: https://github.com/mozilla/publish.webmaker.org/pull/186

### CODE REVIEW

**Comment by Pomax** [comment]
```
> publish.webmaker.org@0.0.1 migrate C:\Users\Mike\Documents\Git projects\collaborations\webmaker-suite\publish.webmaker.org
> knex migrate:latest

Using environment: development
Error: The migration directory is corrupt, the following files are missing: 2_alter_project_table.js
    at validateMigrationList (J:\Junctions\Users\Mike\Documents\Git projects\collaborations\webmaker-suite\publish.webmaker.org\node_modules\knex\lib\migrate\index.js:316:11)
    at processImmediate [as _immediateCallback] (timers.js:374:17)
```

**Comment by Pomax** [comment]
looks like an R+ to me.

File: migrations/4_add_date_tracking_published_projects.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+exports.up = function (knex, Promise) {
+  return Promise.join(
+    knex.schema.table('publishedProjects', function (t) {
+      t.text('_date_created');
```
Comment (jbuck) [inline migrations/4_add_date_tracking_published_projects.js]: This should probably be a timestamp type and not a text type

**Comment by gideonthomas** [comment]
This PR has split into https://github.com/mozilla/publish.webmaker.org/pull/190 and https://github.com/mozilla/publish.webmaker.org/pull/191 so that we can land the migrations first and then the application code.

**Pull request closed without merging**